packages/aws-appsync-subscription-link/package.json

```diff
@@ -19,10 +19,9 @@
     "test-watch": "jest --watch"
   },
   "dependencies": {
-    "apollo-link": "1.2.5",
-    "apollo-link-context": "1.0.11",
-    "apollo-link-http": "1.5.8",
-    "apollo-link-retry": "2.2.7",
+    "@apollo/client": "^3.2.0",
+    "@apollo/link-context": "^2.0.0-beta.3",
+    "@apollo/link-retry": "^2.0.0-beta.3",
```

Comment:
Hey @lukeramsden

I think these two dependencies are not needed since those links are now part of the apollo client package (see https://github.com/apollographql/apollo-client/tree/main/src/link)

Can you remove them and adjust the appropriate imports?

Reply:
@manueliglesias I've removed those 2 deps.